gitconfig

```diff
@@ -11,16 +11,8 @@
   ca = commit --amend -v
   ci = commit -v
   co = checkout
-  co-pr = !sh -c 'git fetch origin pull/$1/head:pr/$1 && git checkout pr/$1' -
-  create-branch = !sh -c 'git push origin HEAD:refs/heads/$1 && git fetch origin && git branch --track $1 origin/$1 && cd . && git checkout $1' -
   ctags = "!sh -c '[ -f .git/hooks/ctags ] || git init; .git/hooks/ctags' git-ctags"
-  current-branch = !sh -c 'git rev-parse --abbrev-ref HEAD' -
```

Comment:
I believe this can be `current-branch = rev-parse --abrev-ref HEAD`? That's what is in my dotfiles. I don't know where the trailing hyphen comes from.